ironfish-cli/src/commands/deposit-all.ts

```diff
@@ -114,11 +94,21 @@ export default class DepositAll extends IronfishCommand {
     Assert.isNotNull(this.client)
     Assert.isNotNull(this.api)
 
+    let fee = null
+
+    if (flags.fee) {
+      const [parsedFee] = BigIntUtils.tryParse(flags.fee)
+
+      if (parsedFee != null) {
+        fee = parsedFee
+      }
+    }
+
     const { canSend, errorReason } = await verifyCanSend(
       this.client,
       this.api,
       expirationSequenceDelta,
-      fee,
+      fee || 1n,
```

Comment:
I think we could remove the `fee` argument from `verifyCanSend` -- checking that the fee is non-negative there seems out-of-place

Reply:
removed fee arugment from verifyCanSend.
Add validation for parsed fee. Check if its valid and > 0